### DISCUSSION TITLE
Switch from CC2531 to Slaesh's CC2652RB stick

### DISCUSSION THREAD

**Discussion by kimmonnens** [Koenkk/zigbee2mqtt#13942] [Devices]
I'm trying to switch from CC2531 to Slaesh's CC2652RB stick for some days now. Until now without any success.
Most of the time I end up with this error: **TypeError: The "path" argument must be of type string. Received undefined**

I am using a PI4 with a 60cm USB cable between the adapter and the PI4. (Only running docker containers on the PI4.)
I am using the latest version of zigbee2mqtt and I have also re-flashed the new stick several times, just to start with a clean stick.
All my hardwired zigbee devices are fully powered off when starting the switch.

In my configuration.yml I have added a new pan_id and network_key. All other configuration is very basic. I have also tried without the pan_id and/or without network_key.

After starting sometimes the "**TypeError: The "path" argument must be of type string. Received undefined"** will be there immediately.
Other times it seems al is started correctly and the front-end is also working fine, but after putting my first device (IKEA remote) in pairing mode, the same error occurs. From then on this error is occurring in the logging every 10 sec or so.

Attached my docker-compose.yml, configuration.yaml and a log file of my last attempt.

I dont' know what I am doing wrong. Any advice is welcome.

[configuration.txt](https://github.com/Koenkk/zigbee2mqtt/files/9534644/configuration.txt)
[docker-compose.txt](https://github.com/Koenkk/zigbee2mqtt/files/9534647/docker-compose.txt)
[zigbee2mqtt.txt](https://github.com/Koenkk/zigbee2mqtt/files/9534651/zigbee2mqtt.txt)

**Comment by kimmonnens** [comment]
Because I was wondering if the Slaesh's CC2652RB stick could be broken I did an extra test. In my Home Assistant I added the Zigbee Home Automation integration. After restarting Home Assistant it automatically found the CC2652RB stick and it suggested the ZHA integration, I only needed to confirm that I wanted to use it. After the configuration was finished I tried to pair the same device (IKEA remote) as on zigbee2mqtt. Pairing completed successfully. I created an automation just for testing the remote. All is working fine, so I assume the Slaesh's CC2652RB stick is working fine.

Because until now I have always used  zigbee2mqtt (with my CC2531), I also would like to use it with the Slaesh's CC2652RB.
So I have no idea why it is not working with zigbee2mqtt and why it is alway giving error "TypeError: The "path" argument must be of type string. Received undefined"

Any advice is welcome.

**Reply by Koenkk** [reply]
Can you clear the `./volumes/zigbee2mqtt/data` (create a backup of it) and try to start z2m again?

**Reply by kimmonnens** [reply]
Before every attempt, I have re-flashed the stick (I don't know if this is needed, but just to be sure) and I always removed all files except the configuration.yaml. Do I also need to remove that file because I thought it was mandatory.
From my docker-compose I removed all services except mosquitto, zigbee2mqtt and portainer, to make this configuration as simple as possible.

**Reply by Koenkk** [reply]
Configuration.yaml is mandatory indeed, how does it look?

**Reply by kimmonnens** [reply]
All files are in the started conversation. Below is the last one I have tried. The config is very basic. I tried with and without network_key and pan_id.

homeassistant: false
permit_join: true
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://mosquitto:1883
serial:
  port: /dev/ttyACM0
frontend: true
advanced:
  log_level: debug
  pan_id: 0x1a64
  network_key: [13, 13, 12, 12, 11, 11, 10, 10, 9, 9, 8, 8, 7, 7, 6, 6]
devices: []

**Reply by Koenkk** [reply]
Can you remove the `devices: []` and start again?
